internal/jimm/model.go

```diff
@@ -553,6 +553,7 @@ func (b *modelBuilder) JujuModelInfo() *jujuparams.ModelInfo {
 // AddModel adds the specified model to JIMM.
 func (j *JIMM) AddModel(ctx context.Context, user *openfga.User, args *ModelCreateArgs) (_ *jujuparams.ModelInfo, err error) {
 	const op = errors.Op("jimm.AddModel")
+	zapctx.Info(ctx, string(op))
```

Comment:
not really, this is the log for the jimm's method entrypoint. 
It can be decoupled from the facade methods, you can potentially call multiple jimm methods as well from a single facade, and also call jimm's methods from rebac-admin.

ex. `CreateModel` facade can call multiple times `jimm.AddModel`, each with the same facade method injected but different entity tag.

Reply:
Fair enough